test/acceptance/replication/replica_replication/endpoints_test.go

```diff
@@ -225,15 +225,15 @@ func (suite *ReplicaReplicationTestSuite) TestReplicationReplicateEndpoints() {
 		}, 30*time.Second, 1*time.Second, "replication operation should be deleted")
 	})
 
-	t.Run("create and delete replication operation", func(t *testing.T) {
+	t.Run("create one op and immediately delete all replication ops", func(t *testing.T) {
 		created, err := helper.Client(t).Replication.Replicate(replication.NewReplicateParams().WithBody(getRequest(t, paragraphClass.Class)), nil)
 		require.Nil(t, err)
 		require.NotNil(t, created)
 		require.NotNil(t, created.Payload)
 		require.NotNil(t, created.Payload.ID)
 		id = *created.Payload.ID
 
-		deleted, err := helper.Client(t).Replication.DeleteReplication(replication.NewDeleteReplicationParams().WithID(id), nil)
+		deleted, err := helper.Client(t).Replication.DeleteAllReplications(replication.NewDeleteAllReplicationsParams(), nil)
```

Comment:
i'm happy you've added this test, i think it will be really important to have this delete all option as an "escape hatch" if a bunch of replica movements are submitted and they all start failing or causing problems. i think it would be useful to add more tests around this in the future as well to increase our confidence that delete all can return a borked cluster to "normal" behavior

Reply:
Definitely! Some e2e tests using the Python client (once it has these ops added to it) that attempts to break the cluster and remove all replication operations would be best imo